### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kmongird**
### Checklist
1. Assign yourself as a reviewer
2. Add at least one tag to your PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,4 +19,6 @@ This is some new text added by Kendall to the README
 
 add text
 
-This is a change..
\ No newline at end of file
+This is a change..
+
```
Comment (SiqiKe) [inline README.md:23]: love this white space

Comment (soo-in) [inline README.md:23]: I am adding a review comment

**Review by SiqiKe — COMMENTED** [review]
hi, see the comment

File: README.md (hunk 1 above)
Comment (kmongird) [reply README.md:23]: Thanks! It's my favorite feature

File: README.md [hunk 2]
```diff
@@ -19,4 +19,6 @@ This is some new text added by Kendall to the README
 
 add text
```
Comment (llturekhankins) [inline README.md:20]: review

**Pull request merged**